StatisticsInR/2.2_descriptive_statistics.Rmd

```diff
@@ -176,22 +250,29 @@ Q4: the maximum value in the set.
 
 Note that the set of data between Q1 and Q3 (which contains the middle 50% of the data) is the interquartile range (IQR).
 
-In R, quartiles may be obtained using the function `quantile()` (NB: this is not a typo, quantile refers to the intervals (0, 25, 50, 75, 100%) at which the quartiles, the values in the data, are assessed). 
+In R, quartiles may be obtained using the function `quantile()`.
+(NB: this is not a typo, quantile refers to the intervals (0, 25, 50, 75, 100%) at which the quartiles, the values in the data, are assessed). 
 If you choose to use only `quantile()` with no other argument than the vector containing the sample, R returns Q0, Q1, Q2, Q3, and Q4. 
 If you need only Q1, Q2 and Q3, you use an extra argument (see the following example):
 
 ```{r quartiles, echo = TRUE}
-x <- c( 21 ,15, 12, 6, 3, -1, -4, -9) # sample vector
-quantile(x) # Find the Q0, Q1, Q2, Q3, and Q4
-quantile(x, probs = c(0.25, 0.5, 0.75)) # Find the Q1, Q2, and Q3 by defining which quartiles (in proportions) using the probs argument
+quantile(height_data2$Height) # Find the Q0, Q1, Q2, Q3, and Q4
+quantile(height_data2$Height, probs = c(0.25, 0.5, 0.75)) # Find the Q1, Q2, and Q3 by defining which quartiles (in proportions) using the probs argument
 ```
 
 Quartiles can be visualized with `ggplot()` and `geom_boxplot()`:
 
```

Comment:
Maybe say they can be vizualized by a boxplot. We will use ggplot and geom_boxplot to do this.